go/websocketbridge/websocket_bridge_test.go

```diff
@@ -0,0 +1,202 @@
+package websocketbridge_test
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"html/template"
+	"log"
+	"net/http"
+	"os"
+	"reflect"
+	"testing"
+
+	"github.com/DATA-DOG/godog"
+	"github.com/DATA-DOG/godog/gherkin"
+	"github.com/Originate/exocom/go/exocom-mock"
+	"github.com/Originate/exocom/go/structs"
+	"github.com/Originate/exocom/go/websocketbridge"
+
+	"golang.org/x/net/websocket"
+)
+
+func newExocom(port int) *exocomMock.ExoComMock {
+	exocom := exocomMock.New()
+	go func() {
+		err := exocom.Listen(port)
+		if err != nil && err != http.ErrServerClosed {
+			log.Fatal(err)
+		}
+	}()
+	return exocom
+}
+
+// Cucumber step definitions
+// nolint gocyclo
+func FeatureContext(s *godog.Suite) {
+	var exocom *exocomMock.ExoComMock
+	var websocketBridgeInstance *websocketbridge.WebsocketBridge
+	var ws *websocket.Conn
```

Comment:
can we please name this `clientWebsocket` instead of `ws` so its purpose is more clear?